packages/terra-cell-grid/src/CellGrid.module.scss

```diff
@@ -0,0 +1,36 @@
+:local {
+  .cell-grid {
+    display: flex;
+    flex: 1 1 auto;
+    margin-bottom: 0;
```

Comment:
Was this margin-bottom property left over from a previous implementation, is it doing anything here on a div?

Reply:
Updated.